react/README.md

```diff
@@ -236,10 +237,38 @@
     });
     ```
 
+## Arrow Functions
+  - When using ES2015 syntax and declaring your components as `class MyComponent extends React.Components {}` you should use arrow functions to define functions that will need to be bound to the components scope. This removes the need to have `.bind()` ran on every render. The arrow function will have the lexical `this` of the component.
+  ```javascript
+  // bad
+  class MyComponent extends React.Component {
+    onLinkClick() {
+      // do stuff
+    }
+
+    render() {
+      return <a onClick={this.onClick.bind(this)}>Clicky</a>;
+    }
+  }
+
+  // good
+  class MyComponent extends React.Component {
+    onLinkClick = () => {
+      // do stuff
+    }
+
+    render() {
+      return <a onClick={this.onClick}>Clicky</a>;
```

Comment:
Nitpick: you declared the method as `onLinkClick` instead of `onClick` (same goes for the bad example)